subprojects/packagefiles/libusb/meson.build

```diff
@@ -0,0 +1,115 @@
+project('libusb', 'c', version: '1.0.26', meson_version: '>=0.60')
+
+add_project_arguments('-D_GNU_SOURCE', language: 'c')
+
+cc = meson.get_compiler('c')
+pconf = import('pkgconfig')
+
+thread_dep = dependency('threads')
+udev_dep = dependency('', required : false)
+macos_dep = dependency('', required: false)
+
+cdata = configuration_data()
+cdata.set('HAVE_CLOCK_GETTIME', cc.has_function('clock_gettime'))
+cdata.set('HAVE_EVENTFD', cc.has_function('eventfd'))
+cdata.set('HAVE_TIMERFD', cc.has_function('timerfd_create'))
+cdata.set('HAVE_SYSLOG', cc.has_function('syslog'))
+cdata.set('HAVE_PIPE2', cc.has_function('pipe2'))
+cdata.set('HAVE_PTHREAD_CONDATTR_SETCLOCK', cc.has_function('pthread_condattr_setclock'))
+cdata.set('HAVE_PTHREAD_SETNAME_NP', cc.has_function('pthread_setname_np'))
+cdata.set('HAVE_PTHREAD_THREADID_NP', cc.has_function('pthread_threadid_np'))
+
+cdata.set('HAVE_ASM_TYPES_H', cc.has_header('asm/types.h'))
+cdata.set('HAVE_NFDS_T', cc.has_header('nfds.h'))
+cdata.set('HAVE_SYS_TIME_H', cc.has_header('sys/time.h'))
+
+cdata.set('HAVE_STRUCT_TIMESPEC', cc.has_header_symbol('time.h', 'struct timespec'))
+
+if cc.get_id() == 'msvc'
+  cdata.set('PRINTF_FORMAT(a, b)', '')
+else
+  cdata.set('PRINTF_FORMAT(a, b)', '__attribute__ ((__format__ (__printf__, a, b)))')
+endif
+
+cdata.set('ENABLE_LOGGING',  get_option('logging'))
+cdata.set('ENABLE_DEBUG_LOGGING',  get_option('debug_logging'))
+cdata.set('USE_SYSTEM_LOGGING_FACILITY',  get_option('system-log'))
```

Comment:
typo in the option, 'systemlog' -> 'system-log'.

Also upstream skips this on Windows and android, and checks for syslog.h, which is only part of POSIX XSI.

Reply:
Oh, you fixed the typo already...